src/server.ts

```diff
@@ -33,27 +33,39 @@ class Honeybadger extends Client {
     }
   ]
 
-  public errorHandler: typeof errorHandler;
-  public requestHandler: typeof requestHandler;
-  public lambdaHandler: typeof lambdaHandler;
+  public errorHandler: typeof errorHandler
+  public requestHandler: typeof requestHandler
+  public lambdaHandler: typeof lambdaHandler
 
-  constructor(opts: Partial<Config> = {}) {
+  config: Config | ServerlessConfig
+
+  constructor(opts: Partial<Config | ServerlessConfig> = {}) {
     super({
       afterUncaught: fatallyLogAndExit,
       projectRoot: process.cwd(),
       hostname: os.hostname(),
       ...opts,
     })
+
+    // serverless defaults
+    const config = this.config as ServerlessConfig
+    config.reportTimeoutWarning = config.reportTimeoutWarning || true
```

Comment:
Bug: I think this should be `??` (which will probably get compiled down to some older syntax). `||` will override the value if it's false.

Reply:
Ah yes, correct. Good catch!